lib/rx-http.js

```diff
@@ -1,33 +0,0 @@
-const httpRx = {};
-const request = require('request');
-const Observable = require('rxjs').Observable;
-const HTTP_STATUS_CODES = require('./http-status-codes');
-const HTTP_METHODS = require('./http-methods');
-const isCorrectHttpMethod = require('./check-http-method');
-
-const isFailedResponse = statusCode => statusCode >= HTTP_STATUS_CODES.BAD_REQUEST;
-const httpRxRequest = (method, url, options) => {
-	if (!isCorrectHttpMethod(method)) {
```

Comment:
Talking about coverage. This if branch is not covered but actually I don't see posibility for user to get in there.